preupg/utils.py

```diff
@@ -565,6 +543,14 @@ def get_preupg_config_file(full_path, key, section="preupgrade-assistant"):
             if config.has_option(section, key):
                 return config.get(section, key)
 
+    @staticmethod
+    def has_ini_file_section(ini_file_path, section):
```

Comment:
I've spoken with @MajoDurco directly. The outcome is that the default/standard configuration file in python has the INI format and is parsed by ConfigParser library. When someone mentions _python_ 
and _config_, IMO most of the time they mean [this python standard format](https://docs.python.org/2/library/configparser.html).